src/Feature_Importance/Feature_Importance.py

```diff
@@ -99,7 +99,7 @@ def select_k_best_features_featurwiz(df, config):
 
     # Run feature selection
     selected_cols = select_k_best_features_sklearn(df, config)
-    # We need label as well for reduced feature used to train data later
+    #   We need label as well for reduced feature used to train data later
```

Comment:
The Reduced_feature.csv from the C# Feature Store will return a Label column that can be used for training. So you won't need to persist a labels.csv separately.

Reply:
I do need this process to create a Selected_Features.csv that has one column with each row being a column name that passed your test.